build.xml

```diff
@@ -5,11 +5,17 @@
   <property name="build.dir" location="build"/>
   <property name="build-src.dir" location="${build.dir}/src"/>
   <property name="generate.dir" location="${build.dir}/generate-src"/>
+  <property name="lib.dir" location="lib"/>
```

Comment:
Guava counts as third-party, so let's follow the third-party rules (third_party/guava directory with an Apache LICENSE file and the Guava JAR file).